test/Yoti.Auth.Tests/ProfileTests.cs

```diff
@@ -29,9 +29,25 @@ public void Profile_GetAttribute_Datetime()
             var attributeValue = new YotiAttributeValue(TypeEnum.Date, Encoding.UTF8.GetBytes(value));
             var initialAttribute = new YotiAttribute<DateTime>(Constants.UserProfile.DateOfBirthAttribute, attributeValue);
 
-            ApplicationProfile applicationProfile = TestTools.Profile.CreateApplicationProfileWithSingleAttribute(initialAttribute);
+            YotiProfile userProfile = TestTools.Profile.CreateUserProfileWithSingleAttribute(initialAttribute);
 
-            YotiAttribute<DateTime> dobAttribute = applicationProfile.GetAttributeByName<DateTime>(Constants.UserProfile.DateOfBirthAttribute);
+            YotiAttribute<DateTime> dobAttribute = userProfile.GetAttributeByName<DateTime>(Constants.UserProfile.DateOfBirthAttribute);
+
+            Assert.AreSame(initialAttribute, dobAttribute);
+            DateTime.TryParseExact(value, "yyyy-MM-dd", CultureInfo.InvariantCulture, DateTimeStyles.None, out DateTime dob);
+            Assert.AreEqual(dob, dobAttribute.GetValue());
+        }
+
+        [TestMethod]
+        public void Profile_GetAttribute_Datetime_Nullable()
+        {
+            string value = "1980-01-13";
+            var attributeValue = new YotiAttributeValue(TypeEnum.Date, Encoding.UTF8.GetBytes(value));
```

Comment:
The _YotiAttributeValue_ wrapper must die!
We can just parse to the appropriate native data type and keep that in the attribute.

Reply:
I've added https://lampkicking.atlassian.net/browse/SDK-680 to remove this